assets/src/edit-story/app/story/effects/usePageBackgrounds.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+/**
+ * Internal dependencies
+ */
+import { createNewElement } from '../../../elements';
+import theme from '../../../theme';
+import { PAGE_WIDTH } from '../../../constants';
+import { MaskTypes } from '../../../masks';
+import createSolidFromString from '../../../utils/createSolidFromString';
+
+// By default, the element should be 33% of the page.
+const DEFAULT_ELEMENT_WIDTH = PAGE_WIDTH / 3;
+
+// Make sure pages have background elements at all times
+function usePageBackgrounds({ currentPage, setBackgroundElement, addElement }) {
+  useEffect(() => {
+    if (!currentPage || currentPage?.backgroundElementId) {
+      return;
+    }
+    const element = createNewElement('shape', {
```

Comment:
If we update the model itself, I think it'd work better in the story reducer rather than a disconnected effect. For one, we wouldn't have history problems with this approach. This might be an important thing to fix right away.